okio/src/commonMain/kotlin/okio/Okio.kt

```diff
@@ -29,4 +29,11 @@ expect fun Source.buffer(): BufferedSource
 expect fun Sink.buffer(): BufferedSink
 
 /** Returns a sink that writes nowhere. */
-expect fun blackholeSink(): Sink
+expect fun blackholeSink(): Sink // expect/actual required for Java interop
```

Comment:
There might be a way to avoid the expect/actual and still conform to the required Java interop but I wasn't able to figure it out. My quick attempt didn't seem to work because of no file merging between this file and the Kotlin/JVM Okio.kt file.